Makefile

```diff
@@ -101,7 +101,11 @@ shellcheck:
 
 .PHONY: test
 test: fetch
+ifeq ($(findstring darwin,$(CARGO_TARGET)),darwin)
+	$(CARGO_TEST) --workspace --exclude linkerd-system
+else
 	$(CARGO_TEST)
+endif
```

Comment:
Presumably a bunch of the other targets suffer from the same problem? I think it's probably better to omit this for now in favor of running cargo directly. If we want to support using the makefile on macOS, we probably need to pursue a more holistic approach.

Reply:
Yeah, it's an issue for many of the targets so this fix wont work. Running cargo manually is totally fine. I was also wondering about the purpose of forcing a compilation error in the system crate - would it be okay to just let it build empty?

Reply:
Backed this out.

Reply:
Yeah, removing the compile error is also probably fine.